internal/cmd/run/apitest.go

```diff
@@ -48,8 +54,9 @@ func runApitest(cmd *cobra.Command, isCLIDriven bool) (int, error) {
 				Dst: os.Stdout,
 			},
 		},
-		Async:         gFlags.async,
-		TunnelService: &restoClient,
+		Async:              gFlags.async,
+		TunnelService:      &restoClient,
+		RequestRateLimiter: rate.NewLimiter(rate.Every(time.Duration(1/queryRequestRate)*time.Second), rateLimitTokenBucket),
```

Comment:
I only realized this now, but I don't think we should pass the RequestRateLimiter all the way from the top inside the runner. That's the wrong domain (separation of concerns).

This ought to be set directly on the `apitestingClient` client.